.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Code-Sleep-Python.iml" filepath="$PROJECT_DIR$/.idea/Code-Sleep-Python.iml" />
```

Comment:
Why all these xml files in the commit ?

Reply:
They are autogenarated. I am using PyCharm to run and work on the python code